server/tests/test_api.py

```diff
@@ -27,6 +27,8 @@ def test_each_cmd_req_rep_has_dedicated_test() -> None:
             for cmd_spec in cmd_specs:
                 cmd_name = cmd_spec["req"]["cmd"]
                 cmd_statuses = [rep_spec["status"] for rep_spec in cmd_spec["reps"]]
+                # Add a fake status corresponding to the tests of HTTP codes 460/461/462
+                cmd_statuses.append("http_46x")
```

Comment:
IMO `http_46x` is not very meaningful: it requires extra knowledge about API and does not say much about what is being tested. Besides, it is also testing `invitation_already_used` (410) and there are TODOs suggesting that `organization_not_found` (404) should also be tested. 

What about something like:
```suggestion
                cmd_statuses.append("http_common_errors")
```
```suggestion
                cmd_statuses.append("common_errors")
```
```suggestion
                cmd_statuses.append("common_parsec_errors")
```

Reply:
I would go with `http_common_errors` then

I think it's useful to mention http and error in the name, so that it hints the related code is in `server/asgi/rpc.py` (i.e. where the HTTP endpoints for the API are defined)